submissions/ai.hackclub.com-based-ai-client/requirements.txt

```diff
@@ -0,0 +1,5 @@
+requests
+json
+os
+time
+copy
```

Comment:
all of these except `requests` are built-in python modules. please remove the others